ts/webui/src/components/nav/Nav.tsx

```diff
@@ -63,7 +63,12 @@ class NavCon extends React.Component<NavProps, NavState> {
             method: 'GET'
         }).then(res => {
             if (res.status === 200) {
-                this.setState({ version: res.data });
+                let formatVersion = res.data;
+                // 2.0 will get 2.0.0 by node, so delete .0 to get real version
+                if (formatVersion.endsWith('.0')) {
```

Comment:
What if `2.0.11`?

Count `.` in version and split on last `.`. Or use regex.

Reply:
why? `2.0.11` is not end with '.0'. endsWith is enough to judge it, i think.